config.env

```diff
@@ -0,0 +1 @@
+ATLAS_URI=mongodb+srv://Hililevitan:Hililevitan7cluster1.mw4wkng.mongodb.net/test
```

Comment:
any way you dont need db to this pr, so dont add it at this PR

Reply:
done